src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -140,6 +140,15 @@ async function writeAccount(req: Request, res: Response): Promise<void> {
     }
     await AdminUsers.services.update(serviceId, serviceUpdateRequest)
     logger.info(`Service ${serviceId} linked to new Gateway Account ${gatewayAccountIdDerived}`)
+
+    logger.info('Service made live', {
```

Comment:
Can we also log `internal` so we can exclude services created for testing purposes?